src/economy/economy.cpp

```diff
@@ -639,14 +695,16 @@ void update_factory_triggered_modifiers(sys::state& state) {
 	});
 }
 
+
 float rgo_effective_size(sys::state const& state, dcon::nation_id n, dcon::province_id p) {
 	auto c = state.world.province_get_rgo(p);
 	if(!c)
 		return 0.01f;
 
-	//Occupied provinces don't employ RGOs
+	float occupation_effect = 1.0f;
+	//Occupied provinces employ less RGOs
 	if(state.world.province_get_nation_from_province_ownership(p) != state.world.province_get_nation_from_province_control(p))
-		return 0.0f;
+		occupation_effect = 0.95f;
```

Comment:
with the occupation effect being this close to 1.0, is it worth having it at all?